### DISCUSSION TITLE
Planning to write blog/articles? Take a look at this.

### DISCUSSION THREAD

**Discussion by kaiwalyakoparkar** [EddieHubCommunity/support#1829] [Ideas] [answered]
This discussion has some tips and suggestions for writing a good blog

**Comment by kaiwalyakoparkar** [comment, marked as answer]
- Choose a platform to write and be consistent with it. Don't make your audience jump over platforms (My recommendation Dev.to or HashNode)
- Try to explain the concept like you are explaining it to a 5 years old kid
- Don't use heavy terms multiple times if the blog is a target for beginners.
- Always give nice and funny examples to explain a term
- Help your audience visualize the concept
- Provide additional resources for them to look after
- Add a relevant title.
- Split your blog/ article into small segments
- Tell your reader why they should consider your writing
- Make them feel good and confident after reading your blog.
- Always reply to comments and questions
- Try to put in extra efforts to search and add images in your blog if necessary

**Reply by ashwinexe** [reply]
Why not medium.com? 🤔

**Reply by ghost** [reply]
> Why not medium.com? 🤔

I don't like medium 🙁

**Reply by kaiwalyakoparkar** [reply]
> Why not medium.com? thinking

Because it does not have much engagement. Whereas the Dev.to and Hashnode has great engagements

**Reply by ashwinexe** [reply]
> > Why not medium.com? 🤔
> 
> I don't like medium 🙁

Haha understandable
